tests/core/providers/test_websocket_provider.py

```diff
@@ -42,12 +45,12 @@
 
 def _mock_ws(provider):
     provider._ws = AsyncMock()
-    provider._ws.closed = False
+    provider._ws.state = State.OPEN
```

Comment:
The `websocket` interface changed, and in the new world, there is a check for `State.OPEN` instead of `conn.closed`. I updated these mock providers so that they more closely match the websocket providers.